Update CHIPS doc (#5051)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/chips/index.md

  Allow developers to opt-in a cookie to "partitioned" storage, with a separate cookie jar per top-level site.
  Partitioned cookies can be set by a third-party service, but only read within the context of the top-level site where they were initially set.
date: 2022-02-15

> updated: 2023-01-26

Author: ```suggestion
updated: 2023-01-27
```

#### First-Party Sets and cookie partitioning

> [First-Party Sets](/blog/first-party-sets-testing-instructions) is a web platform mechanism for developers to declare relationships among sites, so that browsers can use this information to enable limited cross-site cookie access for specific, user-facing purposes. Chrome will use these declared relationships to decide when to allow or deny a site access to their cookies when in a third-party context.

Author: ```suggestion
[First-Party Sets (FPS)](/blog/first-party-sets-testing-instructions) is a web platform mechanism for developers to declare relationships among sites, so that browsers can use this information to enable limited cross-site cookie access for specific, user-facing purposes. Chrome will use these declared relationships to decide when to allow or deny a site access to their cookies when in a third-party context.
```

> First-Party Sets design relies on Storage Access API and does not integrate with CHIPS partitioning.

Reviewer: Can we make this "The current First-Party Sets design ..."

> First-Party Sets design relies on Storage Access API and does not integrate with CHIPS partitioning.

Author: ```suggestion
The current First-Party Sets design relies on Storage Access API and does not integrate with CHIPS partitioning.
```

> If you have a use case that relies on integration of CHIPS and FPS you can (provide examples and feedback on the GitHub issue)[https://github.com/WICG/first-party-sets/issues/94].

Reviewer: 1. Link is not formatted correctly (square brackets/parentheses interchanged).
2. The acronym "FPS" is mentioned for the first time here. Can you add it next to the full spelling earlier in this section?
3. Might be better to say "a shared cookie partition across sites within an FPS" instead of  "integration of CHIPS and FPS".

> If you have a use case that relies on integration of CHIPS and FPS you can (provide examples and feedback on the GitHub issue)[https://github.com/WICG/first-party-sets/issues/94].

Author: ```suggestion
If you have a use case that relies on a shared cookie partition across sites within a FPS you can (provide examples and feedback on the GitHub issue)[https://github.com/WICG/first-party-sets/issues/94].
```

> If you have a use case that relies on integration of CHIPS and FPS you can (provide examples and feedback on the GitHub issue)[https://github.com/WICG/first-party-sets/issues/94].

Author: ```suggestion
If you have a use case that relies on integration of CHIPS and FPS you can [provide examples and feedback on the GitHub issue](https://github.com/WICG/first-party-sets/issues/94).
```
